src/styles/card.scss

```diff
@@ -0,0 +1,101 @@
+.product {
```

Comment:
Double-check all the styles inside your card there is a lot of differences between your card and the design of it in the figma